pygmt/tests/test_clib_put_matrix.py

```diff
@@ -76,7 +76,7 @@ def test_put_matrix_grid():
             with GMTTempFile() as tmp_file:
                 lib.write_data(
                     "GMT_IS_MATRIX",
-                    "GMT_IS_SURFACE",
+                    "GMT_IS_POINT",
```

Comment:
It doesn't seem like we use `lib.write_data` on any user-facing code, or very much in the clib. I know GMT can write an ASCII grid, but maybe we should also test with "GMT_IS_SURFACE" to ensure that:

1. `lib.write_data` can write to a NetCDF properly; and
2. We can use `xarray` to read from that NetCDF file

Reply:
Sounds a good idea, but it needs 6.1.1, so better to add the test in a new pr and only merge that pr after we bump to gmt 6.1.1

Reply:
Ah ok (forgot that we're still on GMT 6.1.0). Leave it until end of the month then.